pysat/_orbits.py

```diff
@@ -456,9 +471,9 @@ def load(self, orbit=None):
                             self.sat.next()
                             self._getBasicOrbit(orbit=1)
                         # check that this orbit should end on the current day
-                        delta = true_date - self.sat.data.index[0]
+                        delta = true_date - self.sat.index[0]
                         # print 'checking if first orbit should land on requested day'
-                        # print self.sat.date, self.sat.data.index[0], delta, delta >= self.orbit_period
+                        # print self.sat.date, self.sat.index[0], delta, delta >= self.orbit_period
```

Comment:
Would it be better to have these print statements as logging statements with a level like 'info' or 'debug'?

Reply:
It certainly would. Logging has been on my wishlist.